### ISSUE TITLE
apache "undefined symbol: ap_proxy_location_reverse_map" load error?

### ISSUE THREAD

**Issue by croensch** [ansible-ThoTeam/nexus3-oss#221]
Following the instructions from the README.md i get the following error (via sysctl):
`apache2: Syntax error on line 146 of /etc/apache2/apache2.conf: Syntax error on line 2 of /etc/apache2/mods-enabled/proxy_http.load: Cannot load /usr/lib/apache2/modules/mod_proxy_http.so into server: /usr/lib/apache2/modules/mod_proxy_http.so: undefined symbol: ap_proxy_location_reverse_map`

[Apparently mods "proxy" and "proxy_http" must be loaded](https://serverfault.com/questions/878641/mod-proxy-http-can-not-be-enabled):
```yaml
    # Debian/Ubuntu only
    - role: geerlingguy.apache
      apache_create_vhosts: no
      apache_mods_enabled: ["proxy.load", "proxy_http.load", "headers.load"]
      apache_remove_default_vhost: true
      tags: apache
```
Works for me.

**Comment by zeitounator** [comment]
Hi.

In [Requirements in README about apache config](https://github.com/ansible-ThoTeam/nexus3-oss/blob/master/README.md#requirements):

> The following modules must be enabled in your configuration: mod_ssl, mod_rewrite, mod_proxy, mod_proxy_http, mod_headers.

In [Example playbook in README](https://github.com/ansible-ThoTeam/nexus3-oss/blob/master/README.md#example-playbook):
```
    # Debian/Ubuntu only
    # - { role: geerlingguy.apache, apache_create_vhosts: no, apache_mods_enabled: ["proxy_http.load", "headers.load"], apache_remove_default_vhost: true, tags: ["geerlingguy.apache"] }
```

If I need to add this info anywhere else in the doc, can you tell me where it is actually missing ? Thanks.

**Comment by zeitounator** [comment]
Ho Sorry, just got it after getting a closer look ! `proxy.load` is missing in the role call.

I'll add that as a doc fix. Thanks a lot for reporting.

**Comment by zeitounator** [comment]
Fixed by merge of #233
